Public/Add-DatabricksNotebookJob.ps1

```diff
@@ -117,11 +117,12 @@ Extended: Simon D'Morias / Data Thirst Ltd
 #>
 
 Function Add-DatabricksNotebookJob {  
-   [cmdletbinding()]
+    [cmdletbinding()]
     param (
         [parameter(Mandatory = $false)][string]$BearerToken,    
         [parameter(Mandatory = $false)][string]$Region,
         [parameter(Mandatory = $true)][string]$JobName,
+        [parameter(Mandatory = $false)]$JobSettings,
```

Comment:
Sorry to be a pain but we have something similar on New-DatabricskCluster:
```
[parameter(ValueFromPipeline)][object]$InputObject
```
Can you name this parameter InputObject and set the ValueFromPipeline - that way we can pipe to it and the naming (albeit stupid) is consistent.

Reply:
Done in [8bf3792](https://github.com/DataThirstLtd/azure.databricks.cicd.tools/pull/125/commits/8bf37920a55319a78072a432dbf6aa2dcac26278)